src/document/utils/file.ts

```diff
@@ -147,10 +148,16 @@ async function loadFromZip<S, A extends Action, L>(
     if (!operationsZip) {
         throw new Error('Operations history not found');
     }
+
     const operations = JSON.parse(
         await operationsZip.async('string'),
     ) as DocumentOperations<A>;
 
+    const operationsError = validateOperations(operations);
+    if (operationsError.length) {
+        console.warn('Invalid operations', operationsError);
```

Comment:
Let's throw the error so it can be dealt with by the caller.

Reply:
added here: [4084461](https://github.com/powerhouse-inc/document-model/pull/34/commits/4084461b3b1465c5152ea6bdb00ea4781330e13a)